cloudigrade/account/tasks.py

```diff
@@ -67,10 +89,47 @@ def copy_ami_snapshot(arn, ami_id, snapshot_region):
     remove_snapshot_ownership.delay(
         arn, customer_snapshot_id, snapshot_region, snapshot_copy_id)
 
+    if reference_ami_id is not None:
+        # If a reference ami exists, that means we have been working with a
+        # copy in here. That means we need to remove that copy and pass the
+        # original reference AMI ID through the rest of the task chain so the
+        # results get reported for that original reference AMI, not our copy.
+        # TODO FIXME Do we or don't we clean up?
+        # If we do, we need permissions to include `ec2:DeregisterImage` and
+        # `ec2:DeleteSnapshot` but those are both somewhat scary...
+        ami_id = reference_ami_id
+
```

Comment:
while I realize that we aren't cleaning up after ourselves at this time, are we going to somehow alert the customer to let them know that we created this copy in their account that they probably want to delete?

Reply:
True… we did say we need to capture some information that can eventually be given back to the end user. I will add a new model and some additional logic around it so we can do this.